_content/tour/ind/struct-types.article

```diff
@@ -1,54 +1,54 @@
-Struct Types
-Struct types are a way of creating complex types that group fields of data together.
+Tipe Struct
+Tipe struct adalah cara mengelompokkan beberapa tipe data ke dalam satu tipe data kompleks.
 
-* Struct Types
+* Tipe Struct
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Tonton Video]]
+- Perlu bantuan finansial, Isi [[https://www.ardanlabs.com/scholarship/][Formulir Beasiswa]]
 
-Struct types are a way of creating complex types that group fields of data together.
-They are a great way of organizing and sharing the different aspects of the data
-your program consumes.
+Tipe struct adalah cara mengelompokkan beberapa tipe data ke dalam satu tipe data kompleks.
+Ini adalah cara yang bagus untuk mengorganisir dan berbagi berbagai aspek data yang dikonsumsi 
+oleh program.
 
-A computer architecture’s potential performance is determined predominantly by its
-word length (the number of bits that can be processed per access) and, more importantly,
-memory size, or the number of words that it can access.
+Potensi dari kinerja suatu arsitektur komputer ditentukan terutama oleh panjang word 
+(jumlah bit yang dapat diproses per akses) dan, yang lebih penting, ukuran memori, 
+atau jumlah word yang dapat diaksesnya.
 
-** Code Review
+** Tinjauan Kode
 
-- *Example* *1:* Declare, create and initialize struct types.
-- *Example* *2:* Anonymous struct types.
-- *Example* *3:* Named vs Unnamed types. 
-- *Example* *4:* Struct type alignments
+- *Contoh* *1:* Mendeklarasikan, membuat, dan menginisialisasi tipe struct.
+- *Contoh* *2:* Tipe struct anonim.
+- *Contoh* *3:* Tipe bernama dan tidak bernama. 
+- *Contoh* *4:* Alignment tipe struct
 
 .play struct-types/example1.go
 .play struct-types/example2.go
 .play struct-types/example3.go
 .play struct-types/example4.go
 
-** Struct and Construction Mechanics
+** Struct dan Mekanisme alignment
```

Comment:
Tipe data struct dan Cara Pembuatannya

Reply:
ack

Reply:
Mekanisme Penyusunan is better in my opinion